css/style.css

```diff
@@ -6,3 +6,79 @@
  *    for example: General styles, Navigation styles, Hero styles, Footer etc.
  * 
  */
+ /* General styles */
+ html {
+    font-family: 'Inconsolata', monospace;
+ }
+
+ body{
+    padding: 50px 40px;
+    background-color: aliceblue;
+ }
```

Comment:
Just something to think about, when using pixels for the placement of things what would happen if the screen size changed or if font size was increased? (Don't think I mentioned in the first call, if I leave a question like this there's no need to reply)